src/shared/components/query/quickSearch/QuickSearch.tsx

```diff
@@ -257,7 +258,8 @@ const Group = (props:any) => {
 
     const groupData:any = props.data.groupData;
 
-    const label = groupData.value + " more " + groupData.type + " (click to load " +
+    const label = groupData.value + " more " + 
+        (groupData.value !== 1 ? groupData.type : Pluralize.singular(groupData.type)) + " (click to load " +
```

Comment:
i think the Pluralize library would actually take care of conditionality too.   So, you could do Pluralize(groupData.type, groupData.value) and it would output the correct thing.

Reply:
see usage documentation https://github.com/blakeembrey/pluralize

Reply:
Oh yes! Thanks for reminding! I have updated this pr.